cmp/relativetimestamp/RelativeTimestamp.js

```diff
@@ -110,6 +126,7 @@ export const relativeTimestamp = elemFactory(RelativeTimestamp);
  * @param {string} [options.emptyResult] - Returned when timestamp is undefined
  */
 export const getRelativeTimestamp = (timestamp, options) => {
+    defaultOptions.short = XH.isMobile;
```

Comment:
See note above

Reply:
can we change this to not be a fat arrow function?  typically use regular function declarations for export